packages/showcase/src/app/components/tab/metadata.ts

```diff
@@ -0,0 +1,514 @@
+
+import { CSS } from "../../../common/html";
+import {
+  simpleModel,
+  withIconsModelMetadata,
+  itemsDisabledModel,
+  itemsDisabledModelMetadata
+} from "./models";
+
+const COMMON_CHILDREN = [
+  {
+    tag: "div",
+    class: CSS.SPACING_BODY.ALL,
+    children: "Item 1 slotted content",
+    properties: [{ name: "slot", value: "item-1" }]
+  },
+  {
+    tag: "div",
+    class: CSS.SPACING_BODY.ALL,
+    children: "Item 2 slotted content",
+    properties: [{ name: "slot", value: "item-2" }]
+  },
+  {
+    tag: "div",
+    class: CSS.SPACING_BODY.ALL,
+    children: "Item 3 slotted content",
+    properties: [{ name: "slot", value: "item-3" }]
+  }
+];
+
+const ICON_UPLOAD_NAME = "ICON_UPLOAD";
+const ICON_UNDO_NAME = "ICON_UNDO";
+const ICON_TOOLS_NAME = "ICON_TOOLS";
+const SLOTS_VARIABLES = [
+  {
+    name: ICON_UPLOAD_NAME,
+    value:
+      "getIconPath({ category: 'system', name: 'upload', colorType: 'primary' })"
+  },
+  {
+    name: ICON_UNDO_NAME,
+    value:
+      "getIconPath({ category: 'system', name: 'undo', colorType: 'primary' })"
+  },
+  {
+    name: ICON_TOOLS_NAME,
+    value:
+      "getIconPath({ category: 'system', name: 'tools', colorType: 'primary' })"
+  }
+];
+
+const SLOTS_COMMON_CHILDREN = {
+  tag: "div",
+  slot: "tab-list-start",
+  class: "buttons-spacer",
+  children: [
+    {
+      tag: "button",
+      class: "button-tertiary button-icon-only",
+      children: [
+        {
+          tag: "ch-image",
+          class: "icon-md",
+          properties: [
+            { name: "src", value: ICON_UPLOAD_NAME, variable: true },
+            { name: "type", value: "mask" }
+          ]
+        }
+      ]
+    },
+    {
+      tag: "button",
+      class: "button-tertiary button-icon-only",
+      children: [
+        {
+          tag: "ch-image",
+          class: "icon-md",
+          properties: [
+            { name: "src", value: ICON_UNDO_NAME, variable: true },
+            { name: "type", value: "mask" }
+          ]
+        }
+      ]
+    },
+    {
+      tag: "button",
+      class: "button-tertiary button-icon-only",
+      children: [
+        {
+          tag: "ch-image",
+          class: "icon-md",
+          properties: [
+            { name: "src", value: ICON_TOOLS_NAME, variable: true },
+            { name: "type", value: "mask" }
+          ]
+        }
+      ]
+    }
+  ]
+};
+export const tabMetadata = {
+  title: "Tab",
+  description:
+    "A tab component is a versatile UI element that organizes content into separate views, each accessible through individual tabs. It enhances user experience by allowing easy navigation between different sections without reloading the page, making it ideal for displaying related content in a compact space.",
+
+  codeSnippets: {
+    listPositionBlockStart: {
+      linkId: "listPosition-block-start",
+      title: "1.1. List Position: Block Start",
```

Comment:
The name for the concept is "Tab List". This name belongs to the [official MDN documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/tab_role#example) and the part name for that element in the `ch-tab-render` is `tab-list`.

```suggestion
      title: "1.1. (Default) Tab List Position: Block Start",
```